### DISCUSSION TITLE
Acceleration of both dp-train and lammps

### DISCUSSION THREAD

**Discussion by J-You** [deepmodeling/deepmd-kit#494] [Q&A] [answered]
Hi all,
I installed v1.3.3 DeepMD-kit by the offline package, only wiht a cpu version. Then, I trained a dp-potential successfully.

I runned patched lammps in DP-kit by the command line:
/home/why/user1/deepmd-kit/bin/mpirun -np 32 /home/why/user1/deepmd-kit/bin/lmp -in in.file

The environment variables were setting according to my machine(2 physical CPUs, each with 18 cores, applying hyper-threading) by:
export OMP_NUM_THREADS=32
export TF_INTRA_OP_PARALLELISM_THREADS=16
export TF_INTER_OP_PARALLELISM_THREADS=2
I also run dp train with this setting.

However, the running speed of lammps seems slow compared with the standard lammps with empirical potential.

Could you please give me some suggestions on how to optimize running acceleration of both dp-train and lammps?

**Comment by njzjz** [comment, marked as answer]
Please do not use both MPI and OpenMP at the same time.

**Reply by J-You** [reply]
Thanks for your response
.
